pkg/controller/registry/resolver/operators.go

```diff
@@ -225,16 +233,25 @@ type OperatorSurface interface {
 	SourceInfo() *OperatorSourceInfo
 	Bundle() *api.Bundle
 	Inline() bool
+	VersionDependencies() []VersionDependency
 }
 
 type Operator struct {
-	name         string
-	replaces     string
-	providedAPIs APISet
-	requiredAPIs APISet
-	version      *semver.Version
-	bundle       *api.Bundle
-	sourceInfo   *OperatorSourceInfo
+	name                string
+	replaces            string
+	providedAPIs        APISet
+	requiredAPIs        APISet
+	version             *semver.Version
+	bundle              *api.Bundle
+	sourceInfo          *OperatorSourceInfo
+	versionDependencies []VersionDependency
```

Comment:
Does this field get populated anywhere? It seems you can only the list of bundles from cache. Then you will need to extract the dependencies information from the bundle (`api.Bundle`).